lib/html.rb

```diff
@@ -33,14 +33,24 @@ def self.run_server(system_description_store, opts)
 EOF
       end
 
+
+
       begin
         setup_output_redirection
-        Server.run!
+        begin
+          Server.run!
+        rescue Errno::EADDRINUSE
+          servefailed_error= <<-EOF.chomp
+Port #{Server.settings.port} is already in use.
+To change the port run: `#{$0} serve --port PORTNUMBER`
```

Comment:
Is this code path also used if you run `machinery show --html`? It that case telling the user to run `machinery serve` would be misleading.